manifests/server/conf.pp

```diff
@@ -154,7 +156,11 @@
 # @param timelimit_hard
 #   Corresponds to ``time.hard`` in ``slapd.conf``
 #
-# @param tlsVerifyClient
+# @param tls_protocol_min
+#   Pin the minimum version of TLS used by the server. This option is only
+#   compatible with openldap-servers >= 2.4.40.
+#
```

Comment:
This parameter,  corresponding to slapd.conf TLSProtocolMin, is very confusing.  I would have set it to 1.2 for TLS 1.2, when it needs to be 3.3.  So, I think an explanation or an example + reference to the slapd.conf man page TLSProtocolMin section would be helpful.